pylint/checkers/refactoring/recommendation_checker.py

```diff
@@ -26,7 +26,7 @@ class RecommendationChecker(checkers.BaseChecker):
             "consider-iterating-dictionary",
             "Emitted when the keys of a dictionary are iterated through the .keys() "
             "method. It is enough to just iterate through the dictionary itself, as "
-            'in "for key in dictionary".',
+            'in "for key in dictionary" or "if key in dictionary".',
```

Comment:
The added text doesn't really match why it's possible. Maybe it would be better to name that as a separate case for this checker? (e.g. dict lookup is quicker than list comparison)